src/components/informationpage/informationpage.scss

```diff
@@ -0,0 +1,98 @@
+@import "../../colors";
+@import "../../frameless";
+
+$ui-secondary: darken($ui-blue, 10%);
+
+#view {
+    padding: 0;
+}
+
+.title-banner {
```

Comment:
Please give the outer `<div>` a className like `information-page`, and then nest everything within this file (other than `#view`) within an `.information-page { ... }` block. This will keep the `InformationPage` styles specific to that component.